docs/vsphere-prerequisites.md

```diff
@@ -1,71 +1,39 @@
+  to work around the pod-to-pod connectivity between hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere)
+* A compatible Windows virtual machine [golden image](vsphere-golden-image.md) must be provided
+* A [DNS entry](#adding-a-dns-entry-in-vsphere-environment) for the internal API endpoint exists in the vSphere
+  environment
 
-## vSphere Windows VM golden image creation
-* Create the VM from an updated version of Windows Server 1909 VM image
-  that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351).
-  This is required to get the OS feature that allows setting the VXLAN UDP port.
-  Please note that this patch is not available for `Windows Server 2019`.
-* Configure SSH on the VM:
-  * You can install OpenSSH Server on the Windows VM by using the provided
-  [powershell script](vsphere_ci/scripts/install-openssh.ps1), or by following the
-  [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
-  * The file, `C:\Users\Administrator\.ssh\authorized_keys` must
-  contain the public key corresponding to the private key in the
-  [secret](https://github.com/openshift/windows-machine-config-operator#usage)
-  on OpenShift.
-  * Make sure the SSH service is running and that it will start on boot.
-* Install and configure VMWare tools on the VM:
-  * Version 11.0.6 or greater is a requirement
-  * The
-  [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-594192DA-0306-425D-B0CD-CB141C4C6874.html)
-  file at `C:\ProgramData\VMware\VMware Tools\tools.conf` has the
-  following entry to ensure that the cloned vNIC
-    generated on the Windows VM by the hybrid-overlay won't be ignored
-    and the VM has an IP Address in vCenter:
-    * `exclude-nics=`
-  * Ensure that the **VMTools** Windows service is running and will
-  start on boot.
-* Pull all the required Windows container base images
-you'll need for your applications. The images you'll pull
-is dependent on the Windows kernel being used. Please see
-[the offical matrix](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-base-images)
-to determine which images you need.
-* Sysprep the VM. Use an `unattend.xml` that will maintain all the
-changes needed for the WMCO. The most important one being that the
-Administrator's home directory stays intact with the SSH public key.
+To create a Windows worker node, you will first need to set up a Windows virtual machine image (the golden image),
+find enclosed an overview of the process required for the [configuration of a Windows VM golden image in 
+vSphere Platform](vsphere-golden-image.md).
```

Comment:
You can remove this and add a link to `vsphere-golden-image.md` in `set up a Windows virtual machine image (the golden image)`